cves/CVE-2011-0722.yml

```diff
@@ -129,7 +139,30 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    This vulnerability existed in the code for nearly two years (Dec 2007 - Nov
+    2009) before being inadvertently resolved. The file was under fairly active
```

Comment:
Was the resolution done deliberately, or was it completely accidentally as the result of some unrelated change?

Reply:
I wouldn't call it _accidental_ per se, but it was indeed inadvertent in that the vulnerability hadn't been discovered yet when the commit was made.

Reply:
I edited this part a bit to make the nature of the fix a bit clearer.